bin/webdrivers/gecko/README.md

```diff
@@ -3,32 +3,66 @@
 New Firefox Marionette driver for Selenium Server Standalone, the browser automation framework.
 
 ## Version
-Version of this release is *v0.11.1*
+Version of this release is *v0.11.1* (https://github.com/mozilla/geckodriver/releases/tag/v0.11.1)
 
 ## Installation
 
-#### Inside your robofile
+#### Inside your robofile (RoboFile.php in the root directory)
+
+You do not have to edit your robofile if you want to use Firefox Gecko / Marionette webdriver.
```

Comment:
Please mention that they have to change the acceptance.suite.yml though (from chrome to firefox) or better:

Just make sure that the browser in your acceptance.suite.yml is set to firefox

Thank you @astridx :-)